cohorts/students_model.js

```diff
@@ -0,0 +1,49 @@
+const knex = require('knex')
+
+const config = require('../knexfile')
+
+const db = knex(config.development)
+
+module.exports = {
+    find,
+    findById,
+    add,
+    update,
+    remove
+}
+
+function find () {
+    return db('students')
+}
+
+function findById (id) {
+    return db('students')
+        .whereIn('cohort_id', id)
```

Comment:
I think the reason why The ID isn't working is here
it's supposed to be `where` not `whereIn`